crates/pathfinder/Cargo.toml

```diff
@@ -40,7 +40,7 @@ serde_with = "1.9.4"
 sha3 = "0.9"
 tempfile = "3"
 thiserror = "1.0.30"
-tokio = "1.11.0"
+tokio = { version = "1.11.0", features = ["test-util"] }
```

Comment:
Just realized this is a non-dev dependency?

Reply:
Lemme revert those unrelated changes and do a separate PR instead.